modules/era/sql/era_droplist.sql

```diff
@@ -5487,22 +5488,22 @@ INSERT INTO `mob_droplist` VALUES (630,0,0,1000,1312,@VRARE);     -- Piece Of An
 
 -- ZoneID: 125 - Desert Beetle
 INSERT INTO `mob_droplist` VALUES (631,0,0,1000,846,@COMMON);   -- Insect Wing (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (631,0,0,1000,889,@COMMON);   -- Beetle Shell (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (631,0,0,1000,894,@UNCOMMON); -- Beetle Jaw (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (631,0,0,1000,889,@UNCOMMON); -- Beetle Shell (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (631,0,0,1000,894,@RARE);     -- Beetle Jaw (Rare, 5%)
 
 -- ZoneID: 114 - Desert Dhalmel
 -- ZoneID: 125 - Desert Dhalmel
-INSERT INTO `mob_droplist` VALUES (632,0,0,1000,857,@COMMON);  -- Dhalmel Hide (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (632,0,0,1000,857,@COMMON);  -- Dhalmel Hide (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (632,0,0,1000,893,@COMMON);  -- Giant Femur (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (632,0,0,1000,938,@COMMON);  -- Sprig Of Papaka Grass (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (632,0,0,1000,4359,@COMMON); -- Slice Of Dhalmel Meat (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (632,0,0,1000,4359,@COMMON); -- Slice Of Dhalmel Meat (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (632,0,0,1000,4359,@VCOMMON); -- Slice Of Dhalmel Meat (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (632,0,0,1000,4359,@VCOMMON); -- Slice Of Dhalmel Meat (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (632,0,0,1000,893,@COMMON);   -- Giant Femur (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (632,0,0,1000,857,@UNCOMMON); -- Dhalmel Hide (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (632,0,0,1000,857,@UNCOMMON); -- Dhalmel Hide (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (632,0,0,1000,938,@RARE);     -- Sprig Of Papaka Grass (Rare, 5%)
```

Comment:
https://ffxiclopedia.fandom.com/wiki/Desert_Dhalmel?so=search
http://ffxidb.com/zones/125/desert-dhalmel

Wiki says its supposed to drop 3 hides, ffxidb says 2.
Drop rates do look consistent with 2 hides @ 10% each. I can check on retail tonight if I can manage to get it to drop 3.

Either way, Fix comment to say Uncommon 10% :D

Reply:
I'm keeping with the 2 hides at uncommon, matches up better with the 5k kills on ffxidb vs 400 on wiki.